Content.Server/Dragon/Components/DragonRiftComponent.cs

```diff
@@ -37,4 +37,15 @@ public sealed partial class DragonRiftComponent : SharedDragonRiftComponent
 
     [ViewVariables(VVAccess.ReadWrite), DataField("spawn", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
     public string SpawnPrototype = "MobCarpDragon";
+
+    //Delta-v Additions begin
```

Comment:
Suggest adding a `bool` just to more easily control the component (in case its used in admemes or whatever), or more easily disable the feature if its too much.

```suggestion
    //Delta-v Additions begin
    [DataField]
    public bool SpawnElites = true;
    
```